docs/guides/deploying_to_google_cloud_platform.md

```diff
@@ -0,0 +1,174 @@
+# Deploying to Google Cloud Platform
+
+Google Cloud Platform provides several ways to deploy an application. For this guide,
+we will walk through how to deploy an application using Google Compute Engine.
+
+For this walkthough, we will accomplish the following:
+
+* Install the Cloud SQL Proxy
+* Deploy the release
+* Connect to Cloud SQL PostgreSQL instance
+* Manage Cloud SQL Proxy and app with systemd
+* Expose the app on port 80
+
+## Prerequisites
+
+You will need to have the performed the following to complete this guide:
+
+  * A running VM in Google Compute Engine with access to the Cloud SQL scope enabled
+    * [Creating a VM on GCE](https://cloud.google.com/compute/docs/instances/create-start-instance)
+  * A Cloud SQL instance
+    * [Creating a Cloud SQL instance](https://cloud.google.com/sql/docs/postgres/create-instance)
+  * Enabled Cloud SQL Admin API
+    * [Enabling and Disabling APIs](https://cloud.google.com/apis/docs/enable-disable-apis)
+  * Cloud SQL instance and GCE instance under the same project
+  * Google Cloud SDK installed and configured
+    * [Configuring the CLI](https://cloud.google.com/sdk/docs/quickstarts)
+  * A release targeted for the VM's operating system
```

Comment:
Could you link to the Walkthrough page here? I just want to make sure people are guided towards the information they need if they land here first for some reason, rather than on the main page.